src/validation.cpp

```diff
@@ -226,6 +226,7 @@ uint256 hashAssumeValid;
 arith_uint256 nMinimumChainWork;
 
 CFeeRate minRelayTxFee = CFeeRate(DEFAULT_MIN_RELAY_TX_FEE);
+CFeeRate minProtocolTxFee = CFeeRate(DEFAULT_MIN_PROTOCOL_TX_FEE);
```

Comment:
Just use COIN should be ok here

Reply:
On second thoughts, it might be better to make two protocol fee rates, something like this:
CFeeRate minProtocolTxFee = CFeeRate(COIN);
CFeeRate minProtocolTxFeeV1 = CFeeRate(CENT);